ZipBuilder/FirebaseSDKs.proto

```diff
@@ -32,6 +32,13 @@ message SDK {
 
   // Whether or not to strip the i386 architecture from the build.
   bool strip_i386 = 8;
+
+  // List of build targets. For internal use only.
+  repeated string build_target = 9;
+
+  // Whether or not to strip both the i386 and armv7 architectures from the
+  // build. For internal use only.
```

Comment:
Add a comment that everything with this flag is built internally, so the ZipBuilder is fine to ignore for its builds.

Reply:
Good call, will do!